website/tosti/api/openapi.py

```diff
@@ -0,0 +1,48 @@
+from oauth2_provider.scopes import get_scopes_backend
+from rest_framework.reverse import reverse
+from rest_framework.schemas.openapi import SchemaGenerator, AutoSchema
+
+
+class OAuthSchemaGenerator(SchemaGenerator):
```

Comment:
Shouldn't this be called the OpenAPISchemaGenerator instead of OAuth (as it is openapi schema)